### DISCUSSION TITLE
software always disconnected

### DISCUSSION THREAD

**Discussion by cabalistljt** [cangzhang/ChampR#289] [General]
in tw server

**Comment by cabalistljt** [comment]
[log.2023-07-17.txt](https://github.com/cangzhang/champ-r/files/12067173/log.2023-07-17.txt)
https://github.com/cangzhang/champ-r/discussions/280
use this ver. and then the log is here

**Comment by cangzhang** [comment]
Can't reproduce this issue, plz make sure to run ChampR (latest version) as admin

![image](https://github.com/cangzhang/champ-r/assets/1357073/354fbce5-fae7-4f74-b9ad-0fcea0dd2b94)

**Reply by cabalistljt** [reply]
is there champr.exe can be placed anywhere?

**Reply by cangzhang** [reply]
yes

**Reply by cabalistljt** [reply]
I have already run champ.exe with admin privileges :(

**Reply by cangzhang** [reply]
Plz check if your anti-virus software were blocking ChampR running powershell scripts.
If the issue still exists, could u post a video or some screenshots to show the detail?

**Reply by cabalistljt** [reply]
It's really caused by Antivirus software,thx very much.

**Reply by cabalistljt** [reply]
another question,why build cannot renew?

**Reply by cangzhang** [reply]
try nightly version here https://github.com/cangzhang/champ-r/actions/workflows/nightly.yml
